testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/component/SampleExternalWebView.kt

```diff
@@ -45,4 +52,20 @@ class SampleWebChromeClient(val context: Context) : WebChromeClient() {
             true
         } else false
     }
+
+    override fun onJsAlert(
+        view: WebView?,
+        url: String?,
+        message: String?,
+        result: JsResult?
+    ): Boolean {
+        val dialog = AlertDialog.Builder(context)
```

Comment:
alternatively, we can show a title e.g. `.setTitle("Error")`

Reply:
@khairul-alam-licon as it looks same as web without the title..and the issue was for removing the title think better not to have a title :)